### ISSUE TITLE
Generate an import tree when building

### ISSUE THREAD

**Issue by pmdevita** [QQuick/Transcrypt#749]
I'm currently working on the python-webpack-loader for using Transcrypt with Webpack. One of Webpack's features is to watch your source files for changes and automatically rebuild when a change occurs. This would be a nice feature to have when using Transcrypt/python-webpack-loader with it as well.

Unfortunately, in order to do this, I need to be able to list every Python file the entry file is dependent on and give it to Webpack so it can watch them for changes. Transcrypt does track this through the compiler's import stack but it discards references once the compilation is complete and there is no way to get this information from the command line.

Could a command line argument be added to get this kind of data? Is there already a way to get this data? Is there a better way to do what I'm trying to do? Thanks!

**Comment by pmdevita** [comment]
Cancelling this, it would be the wrong way to solve this in webpack. What should actually happen is each python file should be compiled one at a time, with each one creating imports that reference other python files rather than already compiled JS files. I'll open another issue for that later.
